README.md

```diff
@@ -903,57 +903,73 @@ Homebrew causes insecure directories like `/usr/local/share/zsh/site-functions`
 
 **Macs with M1 chip**
 
-_January 2021:_ there are no pre-compiled NodeJS binaries for versions prior to 15.x for Apple's new M1 chip (arm64 architecture).
-
-Some issues you may encounter:
-
-- using `nvm` to install, say, `v14.15.4`:
-  - the C code compiles successfully
-  - but crashes with an out of memory error when used
-  - increasing the memory available to node still produces the out of memory errors:
-    ```sh
-    $ NODE_OPTIONS="--max-old-space-size=4096" ./node_modules/.bin/your_node_package
-    ```
-- when using `nvm` to install some versions, the compilation fails
-- after `nvm` successfully compiles some versions, `yarn` or `npm` may later fail to install packages with an `incorrect data check` error.
-
-One solution to this issue is to change the architecture of your shell from arm64 to x86.
-
-Let's assume that:
-- you already have versions `12.20.1` and `14.15.4` installed using `nvm`
-- the current version in use is `14.15.4`
-- you are using the `zsh` shell
-- you have Rosetta 2 installed (macOS prompts you to install Rosetta 2 the first time you open a Intel-only non-command-line application, or you may install Rosetta 2 from the command line with `softwareupdate --install-rosetta`)
-
-```sh
-# Check what version you're running:
-$ node --version
-v14.15.4
-# Check architecture of the `node` binary:
-$ node -p process.arch
-arm64
-# This confirms that the arch is for the M1 chip, which is causing the problems.
-# So we need to uninstall it.
-# We can't uninstall the version we are currently using, so switch to another version:
-$ nvm install v12.20.1
-# Now uninstall the version we want to replace:
-$ nvm uninstall v14.15.4
-# Launch a new zsh process under the 64-bit X86 architecture:
-$ arch -x86_64 zsh
-# Install node using nvm. This should download the precompiled x64 binary:
-$ nvm install v14.15.4
-# Now check that the architecture is correct:
-$ node -p process.arch
-x64
-# It is now safe to return to the arm64 zsh process:
-$ exit
-# We're back to a native shell:
-$ arch
-arm64
-# And the new version is now available to use:
-$ nvm use v14.15.4
-Now using node v14.15.4 (npm v6.14.10)
-```
+Experimental support for the M1 architecture was added in node.js v15.3 and full support was added in v16.0.
+Because of this, if you try to install older versions of node as usual, you will probably experience either compilation errors when installing node or out-of-memory errors while running your code.
+
+So, if you want to run a version prior to v16.0 on an M1 Mac, it may be best to compile
+node targetting the x86_64 Intel architecture so that Rosetta 2 can translate the x86_64
+processor instructions to ARM-based Apple Silicon instructions. Here's what you will need
+to do:
```

Comment:
```suggestion
So, if you want to run a version prior to v16.0 on an M1 Mac, it may be best to compile node targeting the x86_64 Intel architecture so that Rosetta 2 can translate the x86_64 processor instructions to ARM-based Apple Silicon instructions.
Here's what you will need to do:
```